### PR TITLE
[tests] Keep original test-fixture methods naming for easier upgrade

### PR DESCRIPTION
**Pull request by TomasVotruba**
Follow up to https://github.com/rectorphp/rector-src/pull/2889

### CODE REVIEW

File: packages/Testing/PHPUnit/AbstractRectorTestCase.php [hunk 1]
```diff
@@ -83,19 +83,20 @@ protected function tearDown(): void
...
     {
         return FixtureFileFinder::yieldDirectory($directory, $suffix);
     }
 
     /**
-     * @return Iterator<<string>>
+     * @api
+     * @deprecated BC alias for yieldFilesFromDirectory()
      */
     protected function yieldFilePathsFromDirectory(string $directory, string $suffix = '*.php.inc'): Iterator
```
Comment (samsonasik) [inline packages/Testing/PHPUnit/AbstractRectorTestCase.php:97]: the `yieldFilePathsFromDirectory` never exists in 0.14.0 so should can be removed?

Comment (TomasVotruba) [reply packages/Testing/PHPUnit/AbstractRectorTestCase.php:97]: I see. 

Then we'd need to upgrade all our packages to use the `yieldFilesFromDirectory()`. Could you handle it?

Comment (samsonasik) [reply packages/Testing/PHPUnit/AbstractRectorTestCase.php:97]: yes, I will try to update the rector-* first.

Comment (samsonasik) [reply packages/Testing/PHPUnit/AbstractRectorTestCase.php:97]: https://github.com/rectorphp/rector-src/pull/2894

Comment (TomasVotruba) [reply packages/Testing/PHPUnit/AbstractRectorTestCase.php:97]: Awesome work, thank you :+1:

**Pull request merged**